libpod/util.go

```diff
@@ -29,6 +31,93 @@ const (
 	unknownPackage = "Unknown"
 )
 
+// Handle volume options from CLI.
+// Parse "o" option to find UID, GID, Size.
+func VolumeOptions(opts map[string]string) ([]VolumeCreateOption, error) {
```

Comment:
Why is this in Libpod? This looks like CLI parsing code.

Reply:
I this part is shared with `cmd/podman/volume/create` and `libpod` hence it was redundant. I can move this to `parse` package but wanted to prevent cyclic imports, I'll see if it can be moved to a better common place.

Reply:
But why do we need to parse in Libpod? CLI parsing should be done in the frontend

Reply:
This can stay in frontend, I think merging `DriverOpts` and `Options` will allow us to do that. I'll check this.

Reply:
Uh so I checked this again, we need to invoke functions like `WithVolumeSize`, `WithVolumeInodes` , `WithVolumeUID` and `WithVolumeGID` which are not part of parsing but its forcing us to cause redudancy with `VolumeOptions` hence I think it would be better to move this to `libpod` to prevent redundancy and cyclic import.

Reply:
Yeah maybe i can just pass populated `VolumeCreateOption` from frontend to backend no issues.